var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -418,6 +418,8 @@ class PyTensorflow(Package, CudaPackage, ROCmPackage, PythonExtension):
     # and https://github.com/abseil/abseil-cpp/issues/1665
     patch("absl_neon.patch", when="@2.16.1: target=aarch64:")
 
+    # reverting change otherwise the commit patches won't apply
+    patch("revert_fd6b0a4.patch", when="@2.16-rocm-enhanced +rocm")
```

Comment:
Tell me more about this one. Is this commit not needed? Which other patches don't apply? Can we change the order in which the patches are applied? Also see #43040 if it's helpful.

Reply:
The c467913 commit patch won't apply without first reverting fd6b0a4. 
Thanks for the link, I've replaced the patch file now and I'm using the github commit url patch instead.